tests/cypress/integration/actions_tasks_objects/pr_2203_error_сannot_read_property_at_saving_job.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName, labelName } from '../../support/const';
+
+context('Check error сannot read property at saving job', () => {
+
+    const prId = '2203';
+    const createRectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        switchLabel: false,
+        firstX: 100,
+        firstY: 100,
+        secondX: 300,
+        secondY: 300,
+    };
+
+    before(() => {
+        cy.openTaskJob(taskName);
+    });
+
+    after('Remove annotations and save job', () => {
+        cy.removeAnnotations();
+        cy.saveJob();
+    });
+
+    describe(`Testing pr "${prId}"`, () => {
+        it('Create an object in first frame', () => {
+            cy.createRectangle(createRectangleShape2Points);
+        });
+
+        it('Go to next frame and create an object in second frame', () => {
+            cy.get('.cvat-player-next-button').click();
+            cy.createRectangle(createRectangleShape2Points);
+        });
+
+        it('Go to AAM', () => {
+            cy.changeWorkspace('Attribute annotation', labelName);
+        });
+
+        it('Save job and go to previous frame at saving job', () => {
+            cy.saveJob();
+            cy.get('body').type('d');
+        });
+
+        it('Page with the error is missing', () => {
+            cy.wait(100);
```

Comment:
Waiting for a specific time is not the best solution here. Probably need to wait till saving modal window disappears.
This time depends on factor that we cannot control: client-server bandwidth.

Reply:
Thanks! Function cy.wait replaced to wait XHR request success saving.